### ISSUE TITLE
Provide group & description for supplied tasks

### ISSUE THREAD

**Issue by sdavids** [nemerosa/versioning#31]
```
$ gradle tasks

...

:tasks

------------------------------------------------------------
All tasks runnable from root project
------------------------------------------------------------

...

Other tasks
-----------
versionDisplay
versionFile
```

**Comment by dcoraboeuf** [comment]
Available in release 2.2.0. Gone to publication but it can take some time before it is available in the Gradle plugin mirrors.

**Comment by sdavids** [comment]
_group_ should not have "tasks" in it:

```
Versioning tasks tasks
----------------------
versionDisplay - Writes version information on the standard output.
versionFile - Writes version information into a file.
```

**Comment by dcoraboeuf** [comment]
Oops. I was too fast. Fixed in 2.2.1.
Thanks.
